README.md

```diff
@@ -22,19 +22,32 @@ Below are some common usage examples:
 
 ```shell
 # Build images of the hello-world containers
-bake -p hello-world -i 0.0.0-dev
+bake --product hello-world
 
 # Build only one version [0.37.2] of OPA
-bake -p opa=0.37.2 -i 0.0.0-dev
+bake --product opa=0.37.2
+
+# Dry run. Do not buikd anything. Print the the generated Bakefile.
+bake --product hello-world --dry
+
+# Build all OPA images and set the organisation to "sandbox"
+bake --product opa=0.37.2 --organization sandbox
+
+# Build all OPA images and set the image version to a release 24.7.0
+bake --product opa --image-version 24.7.0
 
 # Enable distributed docker cache (requires credentials to access the cache registry)
-bake -p opa --cache
+bake --product opa --cache
+
+# Build the HBase images but use Java 21 instead of the values in conf.py
+# for the java-base and java-devel images.
+bake --product hbase --build-arg 'java-base=21' --build-arg 'java-devel=21'
```

Comment:
Our args are all upper-case, this uses lower case. Is that an error or does casing not matter?

Reply:
`bake` ensures build args are upper cased for Docker/Bakefile.

Reply:
Ah! It'd be great if you could add a sentence about that to the help output.

Reply:
Done https://github.com/stackabletech/image-tools/pull/38/commits/c24672c8157a33bf8f421b1ecf3d2d16ec425268